### DISCUSSION TITLE
Syncing relationship from postgres foreign keys

### DISCUSSION THREAD

**Discussion by basinhr** [Budibase/budibase#13920] [Help] [answered]
It seems very wasteful to have to re-define my relationships again in budibase when I have already defined the foreign keys in postgres. Can budibase simply leverage what I already have?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @basinhr,

I actually do agree with you on this. I've created a feature request for you [here](https://github.com/Budibase/budibase/issues/13955). 

There is likely some explanation for why this doesn't already happen. Budibase may not have the context to correctly define the type of relationship that you are using within your external database. e.g. one -> many, many -> many or one -> one, I suspect this is likely the reason why these need re-defined.

Hopefully this makes sense and thank you for your input.
